### DISCUSSION TITLE
2FA through Mobile App

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#10632] [Mobile] [answered]
I would like to have the possibility to perform a 2FA confirmation via the mobile app.

**Comment by eliperkins** [comment, marked as answer]
You're in luck, @CuzImBisonratte! GitHub Mobile 2FA just shipped this week. You can find [more details in our blog post about it](https://github.blog/2022-01-25-secure-your-github-account-github-mobile-2fa/), as well as [in the GitHub Docs for "Configuring two-factor authentication > Configuring two-factor authentication using GitHub Mobile"](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-github-mobile).

Our team would love to hear your feedback on this new feature. 🙏
